src/price.css

```diff
@@ -0,0 +1,18 @@
+.price {
+  display: flex;
+  align-items: center;
+  margin-top: 23px;
+  margin-bottom: 16px;
+}
+
+.price__text {
+  font-size: 12px;
+  margin-right: 86px;
+  color: var(--secondary-color);
+}
+
+.price_number {
+  color: var(--main-accent-color);
```

Comment:
Ensure that the correct Google font is imported and applied to the 'product-card__price-number' class, as specified in the task description.